From 583d46ba9c4c9bc5fe6c73c9d72e2b1fd0c5c6c5 Mon Sep 17 00:00:00 2001
From: ThirukkumaranVelusamy <thirukkumaranvelusamy@gmail.com>
Date: Thu, 18 Feb 2021 10:49:50 +0530
Subject: [PATCH] patch file added

---
 hello.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/hello.cpp b/hello.cpp
index 8c10515..93a2a05 100644
--- a/hello.cpp
+++ b/hello.cpp
@@ -8,5 +8,6 @@ int main()
 	cout << 5*8 << endl;
 	cout << 5-1 << endl;
 	cout << "Thiru" << endl;
+	cout << "patch file added" << endl;
 	return 0;
 }
-- 
2.30.0

